internal/storage/storage.go

```diff
@@ -384,23 +384,23 @@ func (m *Manager) update(cache *Contract) {
 
 // TODO(jdoak): Change this to return []byte, not string.
 
-func (m *Manager) Serialize() (string, error) {
-	cache := m.Contract()
-
-	serializedCache, err := json.Marshal(cache)
+// Marshal implements cache.Marshaler.
+func (m *Manager) Marshal() ([]byte, error) {
+	b, err := json.Marshal(m.Contract())
```

Comment:
Can we not simply `return json.Marshal(m.Contract())` here?

Reply:
We can since we did the conversion from string returns to bytes, good catch.